fineract-provider/src/main/java/org/apache/fineract/organisation/prequalification/service/PrequalificationWritePlatformServiceImpl.java

```diff
@@ -218,13 +222,13 @@ private List<PrequalificationGroupMember> assembleMembers(JsonCommand command, P
     }
 
     @Override
-    public Long removeFromBlacklist(Long blacklistId) {
-        PrequalificationGroup blacklistClient = this.preQualificationGroupRepository.findById(blacklistId).orElse(null);
-        if (blacklistClient == null) {
-            throw new GroupPreQualificationNotFound(blacklistId);
+    public Long addCommentsToPrequalification(Long groupId, String comment) {
+        PrequalificationGroup prequalificationGroup = this.preQualificationGroupRepository.findById(groupId).orElse(null);
```

Comment:
you could perform the null check at repository wrapper, it makes it reusable.

Same thing you did here centerGroupRepositoryWrapper.findOneWithNotFoundDetection(centerGroupId);

Reply:
Oh yes. thanks. will update that.